docs/INSTALL.md

```diff
@@ -1,130 +1,31 @@
-Please do [check your CNS installation](CNS.md#5-Check-installation) before proceeding.
-
-
-## 1. Clone this repository:
-
-Mind the `--recursive` flag when cloning!
-
-```bash
-git clone --recursive https://github.com/haddocking/haddock3.git
-```
-
-Go to the **haddock3** directory
-
-```bash
-cd haddock3
-```
-
-and type the following commands to first install the [FCC](https://github.com/haddocking/fcc) source code provided
-
-```bash
-cd src/fcc/src
-chmod u+x Makefile
-make
-cd -
-```
-
-Then, follow the same logic to install the [fast-rmsdmatrix](https://github.com/mgiulini/fast-rmsdmatrix) source code
-
-```bash
-cd src/fast-rmsdmatrix/src
-chmod u+x Makefile
-make fast-rmsdmatrix
-cd -
-```
-
-By the end of the above commands, you should be back to the **haddock3**
-main folder.
-
-## 2 Create a virtual environment with Python 3.9+ and install dependencies:
+Create a virtual environment with Python 3.10:
 
 You can use Python's `venv` or `conda` depending on your choice.
 Commands are provided below:
 
 ### with `venv`
 
 ```bash
-virtualenv venv --python=3.9
+virtualenv venv --python=3.10
 source venv/bin/activate
-pip install -r requirements.txt
 ```
 
 ### with `conda`
 
 ```bash
-conda create -n haddock3 python=3.9
+conda create -n haddock3 python=3.10
 conda activate haddock3
-pip install -r requirements.txt
 ```
 
-## 3. Install the HADDOCK3 package and command line clients
-
 ```bash
-python setup.py develop --no-deps
-```
-
-Of note, here we are still using the depreacted `setup.py` command due to HPC, GRID and LOCAL compabilities with CNS.
-
-## 4. Make a CNS binary shortcut to the expected path:
-
-```bash
-mkdir -p bin/
-
-# on mac
-ln -s /PATH/TO/cns_solve_1.3/mac-intel-darwin/source/cns_solve-2206031450.exe bin/cns
-
-# on linux
-ln -s /PATH/TO/cns_solve_1.3/intel-x86_64bit-linux/source/cns_solve-2002171359.exe bin/cns
-```
-
-As long as you have the `(haddock3)` python environment activated, you can
-navigate away from the **haddock3** installation folder. You can run
-HADDOCK3 from anywhere. To run HADDOCK3, follow the [usage
-guidelines](USAGE.md).
-
-
-## 5. Keep your installation up to date
-
-Navigate to the **haddock3** installation folder (the one you cloned from
-GitHub). Ensure you have the `(haddock3)` python environment activated.
-Please keep in mind that HADDOCK3, as well as its
-dependencies, are under active development.
-If the updating processing fails, it is safest to reinstall
-from scratch. Always refer to the latest installation guidelines.
-
-```bash
-# if you used `venv`
-source venv/bin/activate
-
-# if you used `conda`
-conda activate haddock3
-```
-
-Afterwards:
-
-```bash
-# pull the latest source code from our repository to your computer
-git pull
-
-# update the dependencies by running:
-pip install -r requirements.txt  --upgrade
-
-# ensure all command-lines clients are installed
-python setup.py develop --no-deps
```

Comment:
Do we have devel installation instructions somewhere else?

Reply:
I'm waiting for you to add the " developing haddock3 in HPC "  at #988 :)